index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <!-- Toolbar -->
+    <nav class="toolbar">
+        <button type="button"><img src="logo/Icon - Menu.png" alt=""></b>
+    </nav>
```

Comment:
[OPTIONAL]
- [ ] Kindly wrap your `nav` with the `header` tag.

---

- Kindly take a look at the picture which displays the tags that have to be used inside the `body`:
  -  First of all the `header`, which in some cases includes the `nav` inside of it, like our portfolio, and in some cases it doesn't.
  - Secondly, which is not displayed in the picture, the `main` tag, which wraps everything related to your visible content (sections, articles, divs, images, etc...).
  - Last but not least, the `footer`.
  
- All in all, your `body` should consist of 3 tags: `header`, `main`, and `footer`.

    ![image](https://github.com/AliRezaBmeDu/Portfolio-Website/assets/102612396/ded8352e-ecf2-4782-8def-81e7815ccf6c)